src/NewDataList.js

```diff
@@ -209,6 +210,34 @@ module.exports = kind(
 		}
 		this.positionChildren();
 	},
+
+	/**
+	* @private
+	*/
+	refreshThresholds: function () {
+		var tt = this.threshold;
+
+		if (tt) {
+			var
+				v = (this.direction === 'vertical'),
+				val = v ? this.scrollTop : this.scrollLeft,
+				delta = this.delta,
+				cb = this.cachedBounds ? this.cachedBounds : this._getScrollBounds(),
```

Comment:
Does 'scroll threshold' also updated on showing change if we update collection while data list is hidden?